README.md

```diff
@@ -1,2 +1,18 @@
-# winter2022-internships
-List of Winter 2022 internships
+# Winter 2022 Internships by UW CSC
+
+Let's get this show on the road🔥🤩!! This repo has links to tech-related internships positions and external resources as well 💾.
```

Comment:
Nit: Okay LOL we don't need to write the first line of `Let's get this show on the road🔥🤩!`.
Just the second sentence is fineee.

Otherwise, if you wanna keep it here at least space out the emojis
`Let's get this show on the road 🔥 🤩!`